Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:latest
+
+# on-imagebuilder docker build just package the static files into an image.
+# So before docker build the static files must have been built and stored
+# in on-imagebuilder directory. Default folders are common and pxe.
+# You can use your own folders with
+#     docker build --build-arg common=<mysrc> ...
+# Following Dockerfile rules, the <mysrc> path must be inside the context of the build.
+
+ARG common_files=./common/*
+ARG pxe_files=./pxe/*
+
+RUN mkdir -p /RackHD/downloads
+COPY ${pxe_files} /RackHD/downloads/
```

Comment:
All related code snippets are using ```/RackHD/downloads```

change to ```/RackHD/downloads/pxe``` is good but may change too much code.